docs/core/install/runtime.md

```diff
@@ -50,7 +50,7 @@ export PATH=$PATH:$HOME/dotnet
 
 ## Install with a package manager
 
-You can install the .NET Core Runtime with many of the common Linux package managers. For more information, see [Linux Package Manager - Install .NET Core](linux-package-managers.md).
+You can install the .NET Core Runtime with many of the common Linux package managers. For more information, see [Ubuntu 19.10 Package Manager - Install .NET Core](linux-package-manager-ubuntu-1910.md).
```

Comment:
These should be generic. At first there was a generic page but that was removed at some point, however, as this area has grown quite a bit and I think we'll need to reintroduce an overall page.

```suggestion
You can install the .NET Core Runtime with many of the common Linux package managers. For more information, see [Linux Package Manager - Install .NET Core](linux-package-managers.md).
```